server/models/video/video.ts

```diff
@@ -1082,8 +1082,11 @@ export class VideoModel extends Model<VideoModel> {
               '(' +
               'SELECT "video"."id" FROM "video" ' +
               'WHERE ' +
+              'lower(immutable_unaccent("video"."uuid")) % lower(immutable_unaccent(' + escapedSearch + ')) OR ' +
```

Comment:
We want an exact search for the uuid, we don't need the similarity

Reply:
It might be useful if you don't want to type the entire uuid string.

Reply:
I don't think it would be useful, as we use random uuid v4 so a substring/similar uuid has no meaning